Dockerfile

```diff
@@ -1,39 +1,47 @@
 FROM python:3.7-bullseye
-
 WORKDIR /app
 
-# RUN set -x \
-#     && add-apt-repository ppa:mc3man/trusty-media \
-#     && apt-get update \
-#     && apt-get dist-upgrade \
-#     && apt-get install -y --no-install-recommends \
-#         ffmpeg \
-
-# RUN apt-get -y update
-# RUN apt-get -y upgrade
-# RUN apt-get install -y ffmpeg
-
+# Install dependencies
 ENV DEBIAN_FRONTEND=noninteractive
-
 RUN apt-get update && apt-get install -y ffmpeg cmake swig libavcodec-dev libavformat-dev
 RUN ln -s /usr/bin/ffmpeg /usr/local/bin/ffmpeg
 
-COPY . .
+# Copy necessary threatexchange folders
+COPY ./threatexchange/tmk/cpp /app/threatexchange/tmk/cpp
+COPY ./threatexchange/pdq/cpp /app/threatexchange/pdq/cpp
 RUN make -C /app/threatexchange/tmk/cpp
+
+# Copy chromaprint files and build
+COPY ./chromaprint /app/chromaprint
 RUN cd chromaprint && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TOOLS=ON .
 RUN cd chromaprint && make
 RUN cd chromaprint && make install
+
```

Comment:
Should we be removing chromaprint here since the fingerprints will now be computed by Presto?